Panic on PGP verify

Steps to reproduce:

cat > foo.signed

```
-----BEGIN PGP MESSAGE-----
Comment: https://keybase.io/download
Version: Keybase Go 1.0.0 (linux)

xA0DAAoBhxfw9BtMXbgBy+F0AOIAAAAA4mhlbGzhbG/gCgDCwVwEAAEKABAFAlX5
xocJEIcX8PQbTF24AAClshAAUf7rreA8g0OAuCQO9ayGnB83fRj3Hybye3j6u6S3
FEOhqDycfgi6lZbvBPQ/etZp4W7Ci/7w/TJi+7WtrDrZrCA/P1J2b6HGwFYpXU1H
BT3DM4vMQ9+UZO8MVH1HcXFzH0fJ6JKt8oJDg4r2W1yqvsP9tjlRt1EKaRWYsNX0
WKFIXpASYVdBteZkUsDXmSSO3FQI786Ly2xXFitqzQNUVOAakZv60Vc6BrNg8bJI
vcWInEXR8jTBow8xiSBNeYITdM49dXJCwkxn5zQKuQaYcODg3JpiTK037huo8XHP
tiNv4F2rAWWEM3PehL0Tw/BSgoIF6OeagTpeD3ht+vX684f64wfebXzBeeVet3h2
KXyzlc1/wok/kdQuh1W8gTJ7rDECnoKuu3ju0QAmYbg/gSCjyrvsBFYRLu58lSj+
5nUI9vyphU+gcEkg6JW+lFx8gkkwy9yhSukpQHLgWMwSr4o6usi5g8p1cMJeCNA5
hT7Y4nSuv2XG5+0xAfOnKgus5jDJhhgkNzc0SotaV4XGwhog8l7EBISyxZkoDEMt
Kuj/1v6BXLng3HudfZyDrdwpB7NVWZY+YMojKPKqE8dD4QB95cR+LRNiVHAqK3FL
dva5k0lEzptli+jj7D2mBnU9/5BDchISng0aWN0SaTx8gSkRk/ldM26t99ISkhD5
wn4=
=GY62
-----END PGP MESSAGE-----
```

keybase pgp verify < foo.signed

Client output:

```
▶ WARNING Running in devel mode
▶ WARNING key id 9734514050805292472 (8717f0f41b4c5db8) => , 
▶ ERROR EOF from server
```

Server output:

```
keybase --local-rpc-debug-unsafe=csv -d service
10:33:06.959768 ▶ [DEBU keybase json.go:30] 001 + loading config file: /home/cjb/.config/keybase.devel/config.json
10:33:06.959975 ▶ [DEBU keybase json.go:60] 002 - successfully loaded config file
10:33:06.960057 ▶ [DEBU keybase config.go:111] 003 Config: mapping server -> 
10:33:06.960233 ▶ [DEBU keybase config.go:111] 004 Config: mapping socket_file -> 
10:33:06.960357 ▶ [DEBU keybase config.go:111] 005 Config: mapping timers -> 
10:33:06.960388 ▶ [DEBU keybase globals.go:180] 006 Keybase CLI 1.0.0
10:33:06.960420 ▶ [DEBU keybase globals.go:180] 007 - Built with go1.5.1
10:33:06.960449 ▶ [DEBU keybase globals.go:180] 008 - Visit https://keybase.io for more details
10:33:06.960493 ▶ [WARN keybase main.go:46] 009 Running in devel mode
10:33:06.960586 ▶ [DEBU keybase config.go:111] 00a Config: mapping pid_file -> 
10:33:06.960648 ▶ [DEBU keybase flock.go:38] 00b Locked pidfile /run/user/1000/keybased.pid for pid=8119
10:33:06.960687 ▶ [DEBU keybase main.go:216] 00c Locking pidfile /run/user/1000/keybased.pid
10:33:06.960730 ▶ [DEBU keybase config.go:111] 00d Config: mapping socket_file -> 
10:33:06.960778 ▶ [INFO keybase socket.go:47] 00e Binding to unix:/run/user/1000/keybased.sock
10:33:08.561630 ▶ [INFO keybase log.go:117] 00f {unix://@} New connection
10:33:08.561847 ▶ [DEBU keybase log.go:178] 010 {unix://@} serve(0): method=keybase.1.ctl.setLogLevel; err=null; arg=[{"level":2}];
10:33:08.561899 ▶ [DEBU keybase log.go:178] 011 {unix://@} reply(0): method=keybase.1.ctl.setLogLevel; err=null; res=null;
10:33:08.562269 ▶ [DEBU keybase log.go:178] 012 {unix://@} serve(1): method=keybase.1.pgp.pgpVerify; err=null; arg=[{"sessionID":0,"source":{"fd":2},"opts":{"signedBy":"","trackOptions":{"localOnly":false,"bypassConfirm":false},"signature":null}}];
10:33:08.562408 ▶ [DEBU keybase log.go:178] 013 {unix://@} call(0): method=keybase.1.streamUi.read; err=null; arg=[{"sessionID":0,"s":{"fd":2},"sz":4096}];
10:33:08.562587 ▶ [DEBU keybase log.go:178] 014 {unix://@} reply(0): method=keybase.1.streamUi.read; err=null; res="LS0tLS1CRUdJTiBQR1AgTUVTU0FHRS0tLS0tCkNvbW1lbnQ6IGh0dHBzOi8va2V5YmFzZS5pby9kb3dubG9hZApWZXJzaW9uOiBLZXliYXNlIEdvIDEuMC4wIChsaW51eCkKCnhBMERBQW9CaHhmdzlCdE1YYmdCeStGMEFPSUFBQUFBNG1obGJHemhiRy9nQ2dEQ3dWd0VBQUVLQUJBRkFsWDUKeG9jSkVJY1g4UFFiVEYyNEFBQ2xzaEFBVWY3cnJlQThnME9BdUNRTzlheUduQjgzZlJqM0h5YnllM2o2dTZTMwpGRU9ocUR5Y2ZnaTZsWmJ2QlBRL2V0WnA0VzdDaS83dy9USmkrN1d0ckRyWnJDQS9QMUoyYjZIR3dGWXBYVTFICkJUM0RNNHZNUTkrVVpPOE1WSDFIY1hGekgwZko2Skt0OG9KRGc0cjJXMXlxdnNQOXRqbFJ0MUVLYVJXWXNOWDAKV0tGSVhwQVNZVmRCdGVaa1VzRFhtU1NPM0ZRSTc4Nkx5MnhYRml0cXpRTlVWT0Fha1p2NjBWYzZCck5nOGJKSQp2Y1dJbkVYUjhqVEJvdzh4aVNCTmVZSVRkTTQ5ZFhKQ3dreG41elFLdVFhWWNPRGczSnBpVEswMzdodW84WEhQCnRpTnY0RjJyQVdXRU0zUGVoTDBUdy9CU2dvSUY2T2VhZ1RwZUQzaHQrdlg2ODRmNjR3ZmViWHpCZWVWZXQzaDIKS1h5emxjMS93b2sva2RRdWgxVzhnVEo3ckRFQ25vS3V1M2p1MFFBbVliZy9nU0NqeXJ2c0JGWVJMdTU4bFNqKwo1blVJOXZ5cGhVK2djRWtnNkpXK2xGeDhna2t3eTl5aFN1a3BRSExnV013U3I0bzZ1c2k1ZzhwMWNNSmVDTkE1CmhUN1k0blN1djJYRzUrMHhBZk9uS2d1czVqREpoaGdrTnpjMFNvdGFWNFhHd2hvZzhsN0VCSVN5eFprb0RFTXQKS3VqLzF2NkJYTG5nM0h1ZGZaeURyZHdwQjdOVldaWStZTW9qS1BLcUU4ZEQ0UUI5NWNSK0xSTmlWSEFxSzNGTApkdmE1azBsRXpwdGxpK2pqN0QybUJuVTkvNUJEY2hJU25nMGFXTjBTYVR4OGdTa1JrL2xkTTI2dDk5SVNraEQ1CnduND0KPUdZNjIKLS0tLS1FTkQgUEdQIE1FU1NBR0UtLS0tLQ==";
10:33:08.562642 ▶ [DEBU keybase pgp_decrypt.go:61] 015 + PGPDecrypt::Run
10:33:08.562661 ▶ [DEBU keybase pgp_decrypt.go:66] 016 | ScanKeys
10:33:08.562671 ▶ [DEBU keybase scankeys.go:47] 017 + NewScanKeys
10:33:08.562683 ▶ [DEBU keybase login_state.go:801] 018 + Account "LoggedInLoad"
10:33:08.562709 ▶ [DEBU keybase session.go:155] 019 + Loading session
10:33:08.562750 ▶ [DEBU keybase config.go:111] 01a Config: mapping session_file -> 
10:33:08.562765 ▶ [DEBU keybase config.go:111] 01b Config: mapping home -> 
10:33:08.562839 ▶ [DEBU keybase config.go:111] 01c Config: mapping session_file -> 
10:33:08.562854 ▶ [DEBU keybase config.go:111] 01d Config: mapping home -> 
10:33:08.562872 ▶ [DEBU keybase json.go:30] 01e + loading session file: /home/cjb/.cache/keybase.devel/session.json
10:33:08.562910 ▶ [DEBU keybase json.go:60] 01f - successfully loaded session file
10:33:08.562930 ▶ [DEBU keybase session.go:194] 020 Bad 'device_provisioned' value in session file /home/cjb/.cache/keybase.devel/session.json: <root>.device_provisioned: no such key: device_provisioned
10:33:08.562943 ▶ [DEBU keybase session.go:214] 021 - Loaded session
10:33:08.562958 ▶ [DEBU keybase login_state.go:803] 022 - Account "LoggedInLoad"
10:33:08.562968 ▶ [DEBU keybase scankeys.go:49] 023 - NewScanKeys -> %!s(<nil>)
10:33:08.562981 ▶ [DEBU keybase pgp_decrypt.go:72] 024 | PGPDecrypt
10:33:08.562998 ▶ [DEBU keybase pgp_dec.go:53] 025 Calling into openpgp ReadMessage for decryption
10:33:08.563021 ▶ [DEBU keybase scankeys.go:145] 026 ScanKeys:KeysByIdUsage(9734514050805292472, 2) => 0 keys match in memory
10:33:08.563047 ▶ [DEBU keybase api.go:110] 027 + API GET request to http://localhost:3000/_/api/1.0/key/basics.json?pgp_key_id=8717f0f41b4c5db8
10:33:08.563067 ▶ [DEBU keybase api.go:99] 028 | Cli wasn't found; remaking for cookied=false
10:33:08.566876 ▶ [DEBU keybase api.go:180] 029 | Result is: 200 OK
10:33:08.566945 ▶ [DEBU keybase scankeys.go:212] 02a key id 9734514050805292472 (8717f0f41b4c5db8) => , 
10:33:08.566961 ▶ [WARN keybase scankeys.go:214] 02b key id 9734514050805292472 (8717f0f41b4c5db8) => , 
10:33:08.566988 ▶ [WARN keybase scankeys.go:155] 02c error finding keys for 8717f0f41b4c5db8: No public key found
10:33:08.567013 ▶ [DEBU keybase pgp_dec.go:63] 02d message is signed (SignedByKeyId: 9734514050805292472) (have key? false)
10:33:08.567035 ▶ [DEBU keybase pgp_dec.go:70] 02e PGPDecrypt: copied 7 bytes to writer
10:33:08.567054 ▶ [DEBU keybase pgp_decrypt.go:63] 02f - PGPDecrypt::Run %!s(<nil>)
10:33:08.567298 ▶ [DEBU keybase log.go:178] 030 {unix://@} call(1): method=keybase.1.logUi.log; err=null; arg=[{"sessionID":0,"level":4,"text":{"data":"key id 9734514050805292472 (8717f0f41b4c5db8) =\u003e , ","markup":false}}];
panic: runtime error: invalid memory address or nil pointer dereference
[signal 0xb code=0x1 addr=0x0 pc=0x56a2a7]

goroutine 19 [running]:
github.com/keybase/client/go/libkb.PGPDecrypt(0x7f7359152c88, 0xc820182500, 0x7f7359152e90, 0xc8201ac960, 0x7f7359152e58, 0xc82001af60, 0xc82001af60, 0x0, 0x0)
    /home/cjb/gopath/src/github.com/keybase/client/go/libkb/pgp_dec.go:76 +0x8d7
github.com/keybase/client/go/engine.(*PGPDecrypt).Run(0xc8201ae240, 0xc820180240, 0x0, 0x0)
    /home/cjb/gopath/src/github.com/keybase/client/go/engine/pgp_decrypt.go:73 +0x37e
github.com/keybase/client/go/engine.RunEngine(0x7f7359152ce0, 0xc8201ae240, 0xc820180240, 0x0, 0x0)
    /home/cjb/gopath/src/github.com/keybase/client/go/engine/engine.go:64 +0xfd
github.com/keybase/client/go/engine.(*PGPVerify).runAttached(0xc82017c750, 0xc820180240, 0x0, 0x0)
    /home/cjb/gopath/src/github.com/keybase/client/go/engine/pgp_verify.go:95 +0x26c
github.com/keybase/client/go/engine.(*PGPVerify).Run(0xc82017c750, 0xc820180240, 0x0, 0x0)
    /home/cjb/gopath/src/github.com/keybase/client/go/engine/pgp_verify.go:73 +0x11b
github.com/keybase/client/go/engine.RunEngine(0x7f7358109180, 0xc82017c750, 0xc820180240, 0x0, 0x0)
    /home/cjb/gopath/src/github.com/keybase/client/go/engine/engine.go:64 +0xfd
github.com/keybase/client/go/service.(*PGPHandler).PGPVerify(0xc8200301d8, 0x0, 0x2, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, ...)
    /home/cjb/gopath/src/github.com/keybase/client/go/service/pgp.go:101 +0x8cc
github.com/keybase/client/protocol/go.PGPProtocol.func5(0xc8201b2040, 0x0, 0x0, 0x0, 0x0)
    /home/cjb/gopath/src/github.com/keybase/client/protocol/go/keybase_v1.go:2317 +0x1a3
github.com/maxtaco/go-framed-msgpack-rpc/rpc2.(*Request).serve.func2(0xc820176180, 0xc8201b2040, 0x0, 0x0)
    /home/cjb/gopath/src/github.com/maxtaco/go-framed-msgpack-rpc/rpc2/dispatch.go:94 +0x3c
created by github.com/maxtaco/go-framed-msgpack-rpc/rpc2.(*Request).serve
    /home/cjb/gopath/src/github.com/maxtaco/go-framed-msgpack-rpc/rpc2/dispatch.go:104 +0x1d4

goroutine 1 [IO wait]:
net.runtime_pollWait(0x7f7359151f18, 0x72, 0xc820010250)
    /usr/local/go/src/runtime/netpoll.go:157 +0x60
net.(*pollDesc).Wait(0xc82019c0d0, 0x72, 0x0, 0x0)
    /usr/local/go/src/net/fd_poll_runtime.go:73 +0x3a
net.(*pollDesc).WaitRead(0xc82019c0d0, 0x0, 0x0)
    /usr/local/go/src/net/fd_poll_runtime.go:78 +0x36
net.(*netFD).accept(0xc82019c070, 0x0, 0x7f7359151fd8, 0xc82017f620)
    /usr/local/go/src/net/fd_unix.go:408 +0x27c
net.(*UnixListener).AcceptUnix(0xc82017f580, 0xc8200fd9e8, 0x0, 0x0)
    /usr/local/go/src/net/unixsock_posix.go:304 +0x53
net.(*UnixListener).Accept(0xc82017f580, 0x0, 0x0, 0x0, 0x0)
    /usr/local/go/src/net/unixsock_posix.go:314 +0x41
github.com/keybase/client/go/service.(*Service).ListenLoop(0xc82017e640, 0x7f7359150f70, 0xc82017f580, 0x0, 0x0)
    /home/cjb/gopath/src/github.com/keybase/client/go/service/main.go:235 +0x45
github.com/keybase/client/go/service.(*Service).Run(0xc82017e640, 0x0, 0x0)
    /home/cjb/gopath/src/github.com/keybase/client/go/service/main.go:118 +0x312
main.mainInner(0xc8200aa8c0, 0x0, 0x0)
    /home/cjb/gopath/src/github.com/keybase/client/go/keybase/main.go:79 +0x998
main.main()
    /home/cjb/gopath/src/github.com/keybase/client/go/keybase/main.go:32 +0x52

goroutine 17 [syscall, locked to thread]:
runtime.goexit()
    /usr/local/go/src/runtime/asm_amd64.s:1696 +0x1

goroutine 5 [syscall]:
os/signal.loop()
    /usr/local/go/src/os/signal/signal_unix.go:22 +0x18
created by os/signal.init.1
    /usr/local/go/src/os/signal/signal_unix.go:28 +0x37

goroutine 7 [select]:
github.com/keybase/client/go/libkb.(*LoginState).requests(0xc8200fef60)
    /home/cjb/gopath/src/github.com/keybase/client/go/libkb/login_state.go:695 +0x6ed
created by github.com/keybase/client/go/libkb.NewLoginState
    /home/cjb/gopath/src/github.com/keybase/client/go/libkb/login_state.go:99 +0x29f

goroutine 8 [chan receive]:
main.HandleSignals()
    /home/cjb/gopath/src/github.com/keybase/client/go/keybase/main.go:145 +0x10e
created by main.main
    /home/cjb/gopath/src/github.com/keybase/client/go/keybase/main.go:31 +0x44

goroutine 18 [select, locked to thread]:
runtime.gopark(0x12255c0, 0xc820028728, 0x1015338, 0x6, 0x42e218, 0x2)
    /usr/local/go/src/runtime/proc.go:185 +0x163
runtime.selectgoImpl(0xc820028728, 0x0, 0x18)
    /usr/local/go/src/runtime/select.go:392 +0xa64
runtime.selectgo(0xc820028728)
    /usr/local/go/src/runtime/select.go:212 +0x12
runtime.ensureSigM.func1()
    /usr/local/go/src/runtime/signal1_unix.go:227 +0x353
runtime.goexit()
    /usr/local/go/src/runtime/asm_amd64.s:1696 +0x1

goroutine 9 [select]:
stathat.com/c/ramcache.(*Ramcache).cleanup(0xc82001ab40)
    /home/cjb/gopath/src/stathat.com/c/ramcache/ramcache.go:164 +0x12c
created by stathat.com/c/ramcache.New
    /home/cjb/gopath/src/stathat.com/c/ramcache/ramcache.go:44 +0x15c

goroutine 10 [runnable]:
syscall.Syscall(0x0, 0x6, 0xc82016b000, 0x1000, 0x6, 0x1000, 0x0)
    /usr/local/go/src/syscall/asm_linux_amd64.s:18 +0x5
syscall.read(0x6, 0xc82016b000, 0x1000, 0x1000, 0x72, 0x0, 0x0)
    /usr/local/go/src/syscall/zsyscall_linux_amd64.go:783 +0x5f
syscall.Read(0x6, 0xc82016b000, 0x1000, 0x1000, 0xffffffffffffff01, 0x0, 0x0)
    /usr/local/go/src/syscall/syscall_unix.go:160 +0x4d
net.(*netFD).Read(0xc82019c0e0, 0xc82016b000, 0x1000, 0x1000, 0x0, 0x0, 0x0)
    /usr/local/go/src/net/fd_unix.go:228 +0x18b
net.(*conn).Read(0xc820030178, 0xc82016b000, 0x1000, 0x1000, 0x0, 0x0, 0x0)
    /usr/local/go/src/net/net.go:172 +0xe4
bufio.(*Reader).fill(0xc82001ad20)
    /usr/local/go/src/bufio/bufio.go:97 +0x1e9
bufio.(*Reader).ReadByte(0xc82001ad20, 0x10d8040, 0x0, 0x0)
    /usr/local/go/src/bufio/bufio.go:229 +0x7a
github.com/ugorji/go/codec.(*ioDecReader).readn1(0xc8201a8088, 0x404d73)
    /home/cjb/gopath/src/github.com/ugorji/go/codec/decode.go:225 +0x41
github.com/ugorji/go/codec.(*msgpackDecDriver).readNextBd(0xc82000f0e0)
    /home/cjb/gopath/src/github.com/ugorji/go/codec/msgpack.go:566 +0x3a
github.com/ugorji/go/codec.(*msgpackDecDriver).TryDecodeAsNil(0xc82000f0e0, 0x6a8b00)
    /home/cjb/gopath/src/github.com/ugorji/go/codec/msgpack.go:594 +0x2f
github.com/ugorji/go/codec.(*Decoder).decode(0xc8201a8000, 0xc206a0, 0xc8201ac950)
    /home/cjb/gopath/src/github.com/ugorji/go/codec/decode.go:1126 +0x67
github.com/ugorji/go/codec.(*Decoder).Decode(0xc8201a8000, 0xc206a0, 0xc8201ac950, 0x0, 0x0)
    /home/cjb/gopath/src/github.com/ugorji/go/codec/decode.go:1052 +0x74
github.com/maxtaco/go-framed-msgpack-rpc/rpc2.(*Transport).Decode(0xc82019c1c0, 0xc206a0, 0xc8201ac950, 0x0, 0x0)
    /home/cjb/gopath/src/github.com/maxtaco/go-framed-msgpack-rpc/rpc2/transport.go:209 +0x6f
github.com/maxtaco/go-framed-msgpack-rpc/rpc2.(*Packetizer).getFrame(0xc82017f680, 0x39b, 0x0, 0x0)
    /home/cjb/gopath/src/github.com/maxtaco/go-framed-msgpack-rpc/rpc2/packetizer.go:19 +0xb6
github.com/maxtaco/go-framed-msgpack-rpc/rpc2.(*Packetizer).packetizeOne(0xc82017f680, 0x0, 0x0)
    /home/cjb/gopath/src/github.com/maxtaco/go-framed-msgpack-rpc/rpc2/packetizer.go:49 +0x2f
github.com/maxtaco/go-framed-msgpack-rpc/rpc2.(*Packetizer).Packetize(0xc82017f680, 0x0, 0x0)
    /home/cjb/gopath/src/github.com/maxtaco/go-framed-msgpack-rpc/rpc2/packetizer.go:57 +0x35
github.com/maxtaco/go-framed-msgpack-rpc/rpc2.(*Transport).run2(0xc82019c1c0, 0x0, 0x0)
    /home/cjb/gopath/src/github.com/maxtaco/go-framed-msgpack-rpc/rpc2/transport.go:125 +0x33
github.com/maxtaco/go-framed-msgpack-rpc/rpc2.(*Transport).run(0xc82019c1c0, 0x647000, 0x0, 0x0)
    /home/cjb/gopath/src/github.com/maxtaco/go-framed-msgpack-rpc/rpc2/transport.go:164 +0xf1
github.com/maxtaco/go-framed-msgpack-rpc/rpc2.(*Server).Run(0xc82004ff30, 0x7f735914fe00, 0x0, 0x0)
    /home/cjb/gopath/src/github.com/maxtaco/go-framed-msgpack-rpc/rpc2/server.go:18 +0x39
github.com/keybase/client/go/service.(*Service).Handle(0xc82017e640, 0x7f7359152000, 0xc820030178)
    /home/cjb/gopath/src/github.com/keybase/client/go/service/main.go:84 +0x620
created by github.com/keybase/client/go/service.(*Service).ListenLoop
    /home/cjb/gopath/src/github.com/keybase/client/go/service/main.go:238 +0xa4

goroutine 14 [IO wait]:
net.runtime_pollWait(0x7f7359151d98, 0x72, 0xc820010250)
    /usr/local/go/src/runtime/netpoll.go:157 +0x60
net.(*pollDesc).Wait(0xc82019d250, 0x72, 0x0, 0x0)
    /usr/local/go/src/net/fd_poll_runtime.go:73 +0x3a
net.(*pollDesc).WaitRead(0xc82019d250, 0x0, 0x0)
    /usr/local/go/src/net/fd_poll_runtime.go:78 +0x36
net.(*netFD).Read(0xc82019d1f0, 0xc820204000, 0x1000, 0x1000, 0x0, 0x7f735914b050, 0xc820010250)
    /usr/local/go/src/net/fd_unix.go:232 +0x23a
net.(*conn).Read(0xc8200302e8, 0xc820204000, 0x1000, 0x1000, 0x0, 0x0, 0x0)
    /usr/local/go/src/net/net.go:172 +0xe4
net/http.noteEOFReader.Read(0x7f7359153220, 0xc8200302e8, 0xc820132c08, 0xc820204000, 0x1000, 0x1000, 0x0, 0x0, 0x0)
    /usr/local/go/src/net/http/transport.go:1370 +0x67
net/http.(*noteEOFReader).Read(0xc8201af0e0, 0xc820204000, 0x1000, 0x1000, 0x0, 0x0, 0x0)
    <autogenerated>:126 +0xd0
bufio.(*Reader).fill(0xc82001b920)
    /usr/local/go/src/bufio/bufio.go:97 +0x1e9
bufio.(*Reader).Peek(0xc82001b920, 0x1, 0x0, 0x0, 0x0, 0x0, 0x0)
    /usr/local/go/src/bufio/bufio.go:132 +0xcc
net/http.(*persistConn).readLoop(0xc820132bb0)
    /usr/local/go/src/net/http/transport.go:876 +0xf7
created by net/http.(*Transport).dialConn
    /usr/local/go/src/net/http/transport.go:685 +0xc78

goroutine 15 [select]:
net/http.(*persistConn).writeLoop(0xc820132bb0)
    /usr/local/go/src/net/http/transport.go:1009 +0x40c
created by net/http.(*Transport).dialConn
    /usr/local/go/src/net/http/transport.go:686 +0xc9d

goroutine 20 [chan receive]:
github.com/maxtaco/go-framed-msgpack-rpc/rpc2.(*Dispatch).Call(0xc82000f180, 0x10d7dc0, 0x13, 0xc4adc0, 0xc8201af160, 0x0, 0x0, 0x1223980, 0x0, 0x0)
    /home/cjb/gopath/src/github.com/maxtaco/go-framed-msgpack-rpc/rpc2/dispatch.go:141 +0x656
github.com/maxtaco/go-framed-msgpack-rpc/rpc2.(*Client).Call(0xc8201ac180, 0x10d7dc0, 0x13, 0xc4adc0, 0xc8201af160, 0x0, 0x0, 0x0, 0x0)
    /home/cjb/gopath/src/github.com/maxtaco/go-framed-msgpack-rpc/rpc2/client.go:17 +0xe0
github.com/keybase/client/protocol/go.LogUiClient.Log(0x7f7359152548, 0xc8201ac180, 0x0, 0x4, 0xc8201f02c0, 0x33, 0x0, 0x0, 0x0)
    /home/cjb/gopath/src/github.com/keybase/client/protocol/go/keybase_v1.go:1598 +0x195
github.com/keybase/client/go/service.(*LogUI).Log(0xc8201ac1c0, 0x4, 0x1108ee0, 0x1a, 0xc82017a4c0, 0x4, 0x4)
    /home/cjb/gopath/src/github.com/keybase/client/go/service/log.go:23 +0xd3
created by github.com/keybase/client/go/logger.(*Standard).logToExternalLoggers
    /home/cjb/gopath/src/github.com/keybase/client/go/logger/standard.go:352 +0x18d

goroutine 21 [runnable]:
github.com/keybase/client/go/service.(*LogUI).Log(0xc8201ac1c0, 0x4, 0x111c000, 0x20, 0xc8201b2940, 0x2, 0x2)
    /home/cjb/gopath/src/github.com/keybase/client/go/service/log.go:14
created by github.com/keybase/client/go/logger.(*Standard).logToExternalLoggers
    /home/cjb/gopath/src/github.com/keybase/client/go/logger/standard.go:352 +0x18d
```

--- a/go/libkb/pgp_dec.go
+++ b/go/libkb/pgp_dec.go
@@ -73,7 +73,9 @@ func PGPDecrypt(source io.Reader, sink io.Writer, kr openpgp.KeyRing) (*Signatur
 	if md.IsSigned {
 		status.IsSigned = true
 		status.KeyID = md.SignedByKeyId
-		status.SignatureTime = md.Signature.CreationTime
+		if md.Signature != nil {
+			status.SignatureTime = md.Signature.CreationTime
+		}
 		if md.SignedBy != nil {
 			status.Entity = md.SignedBy.Entity
 		}

